packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -232,25 +229,37 @@ describe('<RadioGroup />', () => {
     });
 
     it('should warn when switching from controlled to uncontrolled', () => {
-      const wrapper = mount(
-        <RadioGroup value="foo">
-          <Radio value="foo" />
-        </RadioGroup>,
-      );
-      wrapper.setProps({ value: undefined });
+      let wrapper;
+
+      act(() => {
```

Comment:
Did this fail without the `act`? `enzyme` should already do this. Might just be we use an older `enzyme` version.

Reply:
Yeah, I can test a newer version of enzyme later.

Reply:
Sure. Last time I tried updating it failed some test so this should definitely be done in a different PR.

Reply:
Only just realised its failing with `act`. It worked locally 🙄

Reply:
Hmm, I tried running the test with enzyme 3.9.0 but it still doesn't pass.